components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -33,34 +33,32 @@ bool read_paths_from_file(
         std::string const& input_path_list_file_path,
         std::vector<std::string>& path_destination
 ) {
-    FileReader reader;
-    auto error_code = reader.try_open(input_path_list_file_path);
-    if (ErrorCodeFileNotFound == error_code) {
-        SPDLOG_ERROR(
-                "Failed to open input path list file {} - file not found",
-                input_path_list_file_path
-        );
-        return false;
-    } else if (ErrorCodeSuccess != error_code) {
-        SPDLOG_ERROR("Error opening input path list file {}", input_path_list_file_path);
-        return false;
-    }
-
-    std::string line;
-    while (true) {
-        error_code = reader.try_read_to_delimiter('\n', false, false, line);
-        if (ErrorCodeSuccess != error_code) {
-            break;
-        }
-        if (false == line.empty()) {
-            path_destination.push_back(line);
+    try {
+        clp::FileReader reader(input_path_list_file_path);
+        std::string line;
+        clp::ErrorCode error_code;
+        while (true) {
+            error_code = reader.try_read_to_delimiter('\n', false, false, line);
+            if (clp::ErrorCode::ErrorCode_Success != error_code) {
+                break;
+            }
+            if (false == line.empty()) {
+                path_destination.push_back(line);
```

Comment:
```suggestion
                path_destination.emplace_back(std::move(line));
```
May as well while we're touching this code anyway.